govc/library/info.go

```diff
@@ -66,8 +61,7 @@ func (cmd *info) Register(ctx context.Context, f *flag.FlagSet) {
 	f.BoolVar(&cmd.long, "l", false, "Long listing format")
 	f.BoolVar(&cmd.link, "L", false, "List Datastore path only")
 	f.BoolVar(&cmd.url, "U", false, "List pub/sub URL(s) only")
-
-	cmd.names = make(map[string]string)
+	f.BoolVar(&cmd.stor, "s", false, "Include file item storage details")
```

Comment:
It appears that this arg only takes effect when the cmd is called with a specific library item file (e.g. `library.info -s <library-name>` or `library.info -s <library-name>/<item-name>` would not contain any storage details). Should we clarify this in the help message to avoid confusion?

Reply:
Right, yes, updated the doc